code/_onclick/hud/radial.dm

```diff
@@ -165,20 +174,15 @@ GLOBAL_LIST_EMPTY(radial_menus)
 	E.choice = null
 	E.next_page = FALSE
 
-/datum/radial_menu/proc/SetElement(obj/screen/radial/slice/E, choice_id, angle, anim, anim_order)
+/datum/radial_menu/proc/SetElement(obj/screen/radial/slice/E, choice_id, angle)
 	//Position
 	var/py = round(cos(angle) * radius) + py_shift
 	var/px = round(sin(angle) * radius)
```

Comment:
Could you just merge these into the assignments of pixel_y and pixel_x below this?